support-models/src/main/scala/com/gu/support/workers/states/CreateZuoraSubscriptionState.scala

```diff
@@ -56,12 +58,14 @@ object CreateZuoraSubscriptionProductState {
       firstDeliveryDate: LocalDate,
       appliedPromotion: Option[AppliedPromotion],
       salesForceContact: SalesforceContactRecord,
+      similarProductsConsent: Boolean,
   ) extends CreateZuoraSubscriptionProductState
 
   case class GuardianAdLiteState(
       product: GuardianAdLite,
       paymentMethod: PaymentMethod,
       salesForceContact: SalesforceContactRecord,
+      similarProductsConsent: Boolean,
```

Comment:
Does it make sense to include this for products which won't ever have it?

Reply:
Not sure, it's a good point. I'll remove it for now